client/gutenberg/extensions/map-block/config.js

```diff
@@ -0,0 +1,338 @@
+/**
+ * Wordpress dependencies
+ */
+
+import { __ } from '@wordpress/i18n';
+
+/**
+ * External dependencies
+ */
+
+ /**
+ * Internal dependencies
+ */
+
+export const CONFIG = {
```

Comment:
If we want to be consistent with how Gutenberg core do things, we could name this `settings`:

https://github.com/WordPress/gutenberg/blob/e7d674dc5ee245a41a8c2e9fac88a021dbaded32/packages/block-library/src/audio/index.js#L16

Reply:
Done: https://github.com/Automattic/wp-calypso/pull/27836/commits/636f5c882802e7d50f02eeb9d32f8bf46bade81d